### PR TITLE
Add Persepolis Download Manager

### PR DESCRIPTION
**Pull request by Retia-Adolf**

### CODE REVIEW

**Comment by Ash258** [comment]
It's innosetup [installer](https://i.imgur.com/zeqnEwN.png)
Change it to `"innosetup": true`

You can use `$underscoreVersion` inside autoupdate

I would replace that spaces inside checkver regex with `\\s+`

**Comment by Ash258** [comment]
```json
{
    "homepage": "https://persepolisdm.github.io/",
    "description": "A Free as libre download manager & a GUI for aria2, written in Python",
    "version": "3.1.0",
    "license": "GPL-3.0-only",
    "architecture": {
        "64bit": {
            "url": "https://github.com/persepolisdm/persepolis/releases/download/3.1.0/persepolis_3_1_0_windows_64bit.exe",
            "hash": "fe1fc842b49daa555af3d1ffce4f8f5a2b664620d5004b96cbb6f177ed7be29f"
        },
        "32bit": {
            "url": "https://github.com/persepolisdm/persepolis/releases/download/3.1.0/persepolis_3_1_0)_windows_32bit.exe",
            "hash": "dde3b8b72dc903b369b95af641454f6fe74e8961cd2008fdc5a1427e88b39809"
        }
    },
    "innosetup": true,
    "bin": [
        [
            "Persepolis Download Manager.exe",
            "persepolis"
        ]
    ],
    "shortcuts": [
        [
            "Persepolis Download Manager.exe",
            "Persepolis Download Manager"
        ]
    ],
    "checkver": {
        "github": "https://github.com/persepolisdm/persepolis"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/persepolisdm/persepolis/releases/download/$version/persepolis_$underscoreVersion_windows_64bit.exe",
                "hash": {
                    "url": "https://github.com/AdmiringWorm/chocolatey-packages/raw/master/automatic/persepolis/legal/VERIFICATION.txt",
                    "find": "\\s+checksum64:\\s+([A-f\\d]{64})"
                }
            },
            "32bit": {
                "url": "https://github.com/persepolisdm/persepolis/releases/download/$version/persepolis_$underscoreVersion)_windows_32bit.exe",
                "hash": {
                    "url": "https://github.com/AdmiringWorm/chocolatey-packages/raw/master/automatic/persepolis/legal/VERIFICATION.txt",
                    "find": "\\s+checksum32:\\s+([A-f\\d]{64})"
                }
            }
        }
    }
}

```

**Comment by Retia-Adolf** [comment]
Thanks for your help. I think we'd better not purposely make dependence for Windows app, they intends to be redundant. At least it would be updated with the installer package.

File: persepolis.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/persepolisdm/persepolis/releases/download/3.1.0/persepolis_3_1_0_windows_32bit.exe",
+            "hash": "dde3b8b72dc903b369b95af641454f6fe74e8961cd2008fdc5a1427e88b39809"
+        }
+    },
+    "innosetup": true,
+    "bin": [
+        [
+            "Persepolis Download Manager.exe",
```
Comment (rasa) [inline persepolis.json]: Is it advisable to have spaces in `bin` executables?

Comment (Retia-Adolf) [reply persepolis.json]: I already set non-space alias to it, for convenience to type. But it should not have any issue. If it does, IMHO the OS should be blamed for it, though Windows seems has done better than Unix-like here.

**Pull request merged**